src/wasm-type.h

```diff
@@ -44,6 +44,9 @@ struct Struct;
 struct Array;
 struct Rtt;
 
+enum Nullability { NonNullable, Nullable };
+enum Mutability { NonMutable, Mutable };
```

Comment:
Maybe Immutable? But NonMutable does capitalize better.

Reply:
Oh yeah, if we're going to switch to using meaningful English words, might as well get them right 👍